letsencrypt/constants.py

```diff
@@ -22,6 +22,7 @@
     config_dir="/etc/letsencrypt",
     work_dir="/var/lib/letsencrypt",
     logs_dir="/var/log/letsencrypt",
+    renewal_glob="*.conf",
```

Comment:
Unnecessary. I believe this is a remnant of your CLI option you were using for testing.